src/components/ViewClub.js

```diff
@@ -0,0 +1,361 @@
+/* eslint-disable no-unused-vars */
+import React, { Component } from 'react';
+import {
+    CLUB_ALREADY_EXISTS, UNEXPECTED_ERROR, RESOURCE_NOT_FOUND, CLUB_UPDATED, INVALID_INFO
+} from '../constants';
+import {
+    Grid,
```

Comment:
this is an awful lot of components for 1 single view.

Reply:
Yes it is.